pkg/cmd/repo/view/view.go

```diff
@@ -154,15 +154,15 @@ func viewRun(opts *ViewOptions) error {
 	}
 
 	repoData := struct {
-		FullName    string
-		Description string
-		Readme      string
-		View        string
+		FullName    template.HTML
+		Description template.HTML
+		Readme      template.HTML
+		View        template.HTML
 	}{
-		FullName:    utils.Bold(fullName),
-		Description: description,
-		Readme:      readmeContent,
-		View:        utils.Gray(fmt.Sprintf("View this repository on GitHub: %s", openURL)),
+		FullName:    template.HTML(utils.Bold(fullName)),
+		Description: template.HTML(description),
+		Readme:      template.HTML(readmeContent),
```

Comment:
This looks good! But, I'm wondering whether we need the `html/template` package here? It seems right now we only use the template to concatenate 4 strings with a bit of blank lines in-between—it doesn't seem like the templating approach is buying us much. So one potential fix could be removing the template approach in favor of just outputting these strings in sequence.

If we really wanted to stick with templating, which I think can be promising for the future if we add some kind of color support for it, maybe we should [switch to `text/template`](https://golang.org/pkg/text/template/#Template) instead? Since we're not a HTML app, I don't think we have a need for the features that `html/template` brings over `text/template`

Reply:
Ahh TIL `text/template`! I like that solution and will go that route.

Reply:
Even though the fix is simpler, I think the tests still provide some value in that they ensure we don't mangle some special characters.